### ISSUE TITLE
Merge alert in various cases

### ISSUE THREAD

**Issue by joseluratm** [TheHive-Project/TheHive#942]
# Merge alert in various case at the same time

### Request Type
Feature Request

### Problem Description
I would like to know if you could add the functionality of mergear an alert in several cases.
I enclose an image to make the request more descriptive.

### Complementary information
![casesMarge](https://user-images.githubusercontent.com/7231711/55156868-4f8c9d80-515b-11e9-98f4-5b33dd486d1e.png)

**Comment by zpriddy** [comment]
The would be interesting to be able to launch (make) multiple cases from one alert. However I think in the current framework this would not be possible. Each alert can only be related to one case. 

I think the official answer would be, why cant you merge the three cases into one case? ( I do understand that there would be edge cases where this would not be the right approach.. But i think those are rare.. ) 

I would love to understand the full use case for this.. and how other people would use a feature like this.

**Comment by saadkadhi** [comment]
Who summoned The Official Voice™? :-)

Indeed, in our current design of TheHive, an alert can either be ignored or transformed into a case (i.e. an investigation). And since TheHive does not support multi-tenancy yet, we need to understand the use case where an alert can become two or three distinct investigations before considering your feature request.

**Comment by zpriddy** [comment]
Let me rephrase that now that I'm awake.. the current supported implementation would be... 😄

**Comment by joseluratm** [comment]
Hi,

When we are talking about indicators of compromise the relation alert - case must be unique because this activity is relative to unique investigation (especially for decision making).

Instead, when we are working with observables (not IoCs) maybe my alert have information that i would like to put separately but correlated.

For example, if i have an opened investigation with the observable A (and a lot more) , and other investigation with the observable B (and a lot more). I think it could great to have capatibilities to save a new alert (Imagine, this alert have observable A and observable B) in both investigations. Why i think this could be great? Because until we are sure that Case A and Case B have heavy relation, i think its important to have the observable A in case A and observable B in case B but for correlating (maybe) observable B in case A and observable A in case B serves to have them related. 

Now this is not possible and then i must have a case A with observable A and case B with observable B and observable A when i think this cases have bidirectionally related

Thanks!!!

**Comment by torsolaso** [comment]
here one more interesting in this feature. I think it would be great to manage alerts.
